### PR TITLE
Added resource to error logs

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
When errors are logged they don't contain enough context to be useful.
This PR adds the resource that was being requested when the error occurred.
We were seeing many 404 errors but the logs did not show what resource was being requested.

### How to review 
Hit an invalid url and observe that the resource you requested is in the logged error

### CODE REVIEW

**Review by benjefferies — APPROVED** [review]

**Review by benjefferies — CHANGES_REQUESTED** [review]
Should probably add to the else statement too?

File: app/views/errors.py [hunk 1]
```diff
@@ -30,46 +30,46 @@ def add_cache_control(response):
...
 def service_unavailable(error=None):
-    log_exception(error)
+    log_exception(error, 503)
     return _render_error_page(503)
 
 
 @errors_blueprint.app_errorhandler(MultipleSurveyError)
 def multiple_survey_error(error=None):
-    log_exception(error)
+    log_exception(error, 200)
```
Comment (benjefferies) [inline app/views/errors.py:58]: Seems weird to return 200 here, maybe a 409 would be more applicable

Comment (jonnyshaw89) [reply app/views/errors.py:58]: This PR isn't really about changing the status that is returned. Its more adding disability of what is returned. @ajmaddaford what do you think about this?

Comment (ajmaddaford) [reply app/views/errors.py:58]: Agreed, this pull request is to add the url and status code to the log output, if we want to change the behaviour we should do so in another pull request.

**Review by ajmaddaford — APPROVED** [review]

**Review by benjefferies — APPROVED** [review]

**Pull request merged**